app/assets/javascripts/questions.coffee

```diff
@@ -1,3 +1,11 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready ->
+  $('input:radio').click ->
```

Comment:
I know and we do not plan to use default radio-buttons, because they don't look good.